components/MeshRF.vue

```diff
@@ -3,21 +3,21 @@
     <v-card-title class="headline">
       Mesh RF
       <v-spacer />
-      <v-icon color="blue">mdi-wifi</v-icon>
+      <v-icon color="green">mdi-wifi</v-icon>
     </v-card-title>
     <v-card-text>
       <v-row>
         <v-col class="text-end">
           <p class="label">SSID:</p>
           <p class="label">Channel:</p>
-          <p class="label">Bandwidth:</p>
-          <p class="label">Frequency:</p>
+          <p class="label">Bandwidth (MHz):</p>
+          <p class="label">Frequency (MHz):</p>
```

Comment:
Frequency is returned from the API with the GHz label already.  (It's not in MHz)

Reply:
Perfect!  Thanks.  Just pushed that change.

Reply:
actually, i think it's better to update the api to return the raw freq in MHz.   I just submitted that PR for discussion.  Let's hold this until that is resolved.   If it is approved, then the (MHz) label will be fine.